packages/storage-ui/src/Contexts/StorageContext.tsx

```diff
@@ -102,19 +135,115 @@ const StorageProvider = ({ children }: StorageContextProps) => {
     }
   }, [storageApiClient])
 
-  const refreshPins = useCallback(({ searchedCid = undefined, searchedName = "" }: RefreshPinParams = {}) => {
+  const refreshPins = useCallback(() => {
     storageApiClient.listPins(
-      searchedCid ? [searchedCid] : undefined,
-      searchedName,
+      pinsParams.searchedCID ? [pinsParams.searchedCID] : undefined,
+      pinsParams.searchedName || undefined,
```

Comment:
I guess this should work as well
```suggestion
      pinsParams.searchedName,
```